.gitignore

```diff
@@ -1 +1,3 @@
 kata-agent
+kata-agent.service
+agent
```

Comment:
This line is redundant I think?

Reply:
Because it is  `kata-agent`  to be ignored right? I added because go build also generates that binary (if makefile is not used). So I like to see the binary in git status.

Reply:
Ah - fair point. On the topic of `go build`, once we start getting elaborate, we might want to consider something like:

- https://github.com/clearcontainers/runtime/pull/272#issuecomment-316339910